Update NNS dev docs (#3270)
dfinity/portal: docs/developer-docs/daos/nns/concepts/proposals/proposal-submit.mdx

To send ICP to the account, all you need is to input the `account id` in your wallet of choice.

**Now that the account has the required ICP, spawn a neuron**

Note: This section assumes you have `quill` installed properly.

```
// Craft a message to "stake a the neuron by choosing a 8-character $NAME and an $AMOUNT to stake"
// Note: you need to stake at LEAST 10 ICP to send proposals as $AMOUNT
$ quill --pem-file private.pem neuron-stake --name $NAME --amount $AMOUNT > message.json

//Send the message crafted above.
$ quill send message.json
```

* You should see a message with a neuron ID spawned
* If successful, response will return `neuron ID: 9_218_890_096_040_352_708`. This means the neuron ID is `9218890096040352708`.

**Increase neuron dissolve to at least 6 months**

* Time is measured in seconds in the NNS
* One year in seconds: 31557600 seconds

```
// Craft message to increase dissolve delay of neuron to 1 year
$ quill --pem-file private.pem neuron-manage 9218890096040352708 --additional-dissolve-delay-seconds 31557600 > message.json

//send message created above to the NNS
$ quill send message.json
```

### How to send a proposal

> This sections shows how to create a motion proposal similar to this one: https://dashboard.internetcomputer.org/proposal/42626

Reviewer: ```suggestion
This sections shows how to create a [motion proposal](https://dashboard.internetcomputer.org/proposal/42626).
```

#### Craft a proposal in markdown

> The IC Dashboard and the NNS frontend dapp both render markdown of the `summary` field. It is recommended to use standard markdown for your motion proposal.

Reviewer: ```suggestion
The IC dashboard and the NNS frontend dapp both render markdown of the `summary` field. It is recommended to use standard markdown for your motion proposal.
```

> Example markdown for a NNS motion proposal:

Reviewer: ```suggestion
Below is an example of using markdown for a NNS motion proposal:
```

> ```

Reviewer: ```suggestion
```md
```

## Background

### Relation to the original SNS design & overall approach
DFINITY Foundation presented a first proposed [SNS design](https://www.youtube.com/watch?v=L4443aaAP5A) in October 2021.

## Community conversation & Timeline
The NNS team is ready to work on this feature!
To get the design approved by the community, the following schedule was followed:

* Jan 11-18th: Design discussion on the forum thread
* Jan 18th: Community conversation about the topic
* Jan 20th: Developer discussion on Discord
* today: motion proposal submission proposing the above design

## What is being asked of:

* Vote accept or reject on NNS Motion
* Participate in technical discussions as the motion moves forward
```

#### Send proposal via dfx

> Example structure of sending an NNS motion proposal with placeholders ``$TITLE`, 
> `$MOTION_TEXT`,`$SUMMARY`, and`$NEURON_ID` (these are not meant to be Unix variables and you need to fill them directly within the command):
> 
> * note that `rrkah-fqaaa-aaaaa-aaaaq-cai` is the ID of the Governance Canister on the NNS so this command is is
> sending a message to the `manage_neuron` method of the Governance Canister of the NNS.

Reviewer: ```suggestion
Below is an example structure of sending an NNS motion proposal with placeholders `$TITLE`, 
`$MOTION_TEXT`,`$SUMMARY`, and`$NEURON_ID`. These are not variables and you need to replace them with real values directly within the command:

`rrkah-fqaaa-aaaaa-aaaaq-cai` is the ID of the governance canister on the NNS so this command is sending a message to the `manage_neuron` method of the governance canister of the NNS.
```

```
dfx --identity proposals canister --network ic call rrkah-fqaaa-aaaaa-aaaaq-cai manage_neuron '(record {id = null; command=opt variant {MakeProposal=record {url=""; title=opt "$TITLE";action=opt variant {Motion=record {motion_text="$MOTION_TEXT"}}; summary="

$SUMMARY

"}}; neuron_id_or_subaccount=opt variant {NeuronId=record {id=$NEURON_ID:nat64}}})'
```

> Example NNS motion proposal with`$TITLE`,`$MOTION_TEXT`,`$SUMMARY`, and`$NEURON_ID` filled out:

Reviewer: ```suggestion
Below is an example NNS motion proposal with`$TITLE`,`$MOTION_TEXT`,`$SUMMARY`, and`$NEURON_ID` filled out:
```

## Background

### Relation to the original SNS design & overall approach
DFINITY Foundation presented a first proposed [SNS design](https://www.youtube.com/watch?v=L4443aaAP5A) in October 2021.

## Community conversation & Timeline
The NNS team is ready to work on this feature!
To get the design approved by the community, the following schedule was followed:

* Jan 11-18th: Design discussion on the forum thread
* Jan 18th: Community conversation about the topic
* Jan 20th: Developer discussion on Discord
* today: motion proposal submission proposing the above design

## What is asked of the community:

* Vote accept or reject on NNS Motion
* Participate in technical discussions as the motion moves forward

"}}; neuron_id_or_subaccount=opt variant {NeuronId=record {id=9218890096040352708:nat64}}})'
```

#### Check dashboard to see proposal was successful

> If successful, dfx will return something like this:

Reviewer: ```suggestion
If successful, `dfx` will return output such as:
```

```
(
  record {
    2_171_433_291 = opt variant {
      3_217_030_240 = record {
        2_744_746_248 = opt record { 23_515 = 42_626 : nat64 };
      }
    };
  },
)
```

> where `42_626`  or rather `42626` is the ID of the proposal. Your proposal will have a different ID.

Reviewer: ```suggestion
In this example, `42_626` (or rather `42626`) is the ID of the proposal. Your proposal will have a different ID.
```

> You will be able to see the proposal here: https://dashboard.internetcomputer.org/proposal/42626

Reviewer: ```suggestion
You will be able to see the proposal at the URL `https://dashboard.internetcomputer.org/proposal/42626`.
```

### Best practices for motion proposals

> #### Writing the Proposal

Reviewer: ```suggestion
#### Writing the proposal
```

> The proposal can accept any free-form text. Below is a recommended starting template for anyone starting to think about how to write one:

Reviewer: ```suggestion
The proposal can accept any free form text. Below is a recommended starting template for anyone starting to think about how to write one:
```
